packages/sdk/src/l2-provider.ts

```diff
@@ -7,6 +7,12 @@ import cloneDeep from 'lodash/cloneDeep'
 import { L2Provider, ProviderLike, NumberLike } from './interfaces'
 import { toProvider, toNumber, toBigNumber } from './utils'
 
+// Not exactly sure when the provider wouldn't have a formatter function, but throw an error if
```

Comment:
Comment seems to be copied into the wrong location, should be above the formatter check below